src/views/Dashboard.vue

```diff
@@ -1,10 +1,16 @@
 <template>
+  <header>
+    <h4>Filters</h4>
+    <select @change="handleFilterChange" v-model="genreFilter" class="filter" multiple>
+      <option :value="genre" v-for="genre in $store.getters.getGenres">{{genre.name}}</option>
+    </select>
+  </header>
   <main v-if="$store.getters.getMovies">
-    {{$store.getters.getMovies.length === 0 ? 'There is no movies in database' : ''}}
-    <Movie :movie-data="$store.getters.getMovies[0]" class="featured" />
+    {{!$store.getters.getMovies ? 'There is no movies in database' : ''}}
+    <Movie :movie-data="$store.getters.getFeaturedMovie" class="featured" />
```

Comment:
https://vuex.vuejs.org/guide/getters.html
 pogledaj ovaj deo. Nije preporuceno da unutar componenata getujes iz vuexa nego preko computed metode

Reply:
https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper